src/main/java/core/basesyntax/Plane.java

```diff
@@ -2,4 +2,83 @@
 
 public class Plane {
 
```

Comment:
the empty line after the class declaration is redundant

Reply:
fixed!